server/storage/mvcc/index.go

```diff
@@ -92,9 +96,6 @@ func (ti *treeIndex) keyIndex(keyi *keyIndex) *keyIndex {
 func (ti *treeIndex) visit(key, end []byte, f func(ki *keyIndex) bool) {
```

Comment:
Note about convention. In etcd code we prefix functions with `unsafe` if functions requires external caller to lock mutex. This would mean that function `Get` -> `unsafeGet` and `visit` -> `unsafeVisit`.

Reply:
With function renames it was much clearer for me what this PR is changing. Your right, this PR should not have any performance impact.